src/main/java/dk/kb/ds/cumulus/export/FieldMapper.java

```diff
@@ -111,6 +111,7 @@ public void toXML(XMLStreamWriter xml){
                 xml.writeCharacters(NEWLINE);
             } catch (XMLStreamException e) {
                 log.error("XML not filled properly",e);
+                throw new RuntimeException();
```

Comment:
Please add the `XMLStreamException` when throwing the `RunTimeException`, so that users of the tool can use it for debugging: `throw new RuntimeException(e);`. If you are really service minded, also add a messate stating the `field` and `value` that was being added.